pkg/kubehound/storage/storedb/mongo_provider.go

```diff
@@ -94,11 +132,26 @@ func (mp *MongoProvider) HealthCheck(ctx context.Context) (bool, error) {
 }
 
 func (mp *MongoProvider) Close(ctx context.Context) error {
-	return mp.client.Disconnect(ctx)
+	var res *multierror.Error
```

Comment:
what go version does tnat need?

Reply:
1.20, so from feb 2023

Reply:
got it - I think in this case will leave as is (since we already use this style in a number of other places) and not adding a dep would be the main reason for changing